source/ProtonPack/ProtonPack.ino

```diff
@@ -3087,7 +3237,28 @@ void wandStoppedFiring() {
         // Nothing
       break;
     }
-  }
+  } 
+
+  resetFastLed();
+}
+
+void wandStoppedFiring() {
+  resetFastLed();
+
+  modeFireStopSounds();
+
+  // A tiny ramp down delay helps with the sounds.
+  ms_firing_stop_sound_delay.start(i_fire_stop_sound_delay);
+
+  // Keep this code for later future updates, when using 4 wire led chipsets, the delay timer above can be avoided.
+  //wandStopFiringSounds();
```

Comment:
Ah, now I think I understand--this _might_ be the reason why I would sometimes still hear effects such as the slime loop still playing if I timed my actions right, as this code would not execute at this point. Though I see there's a timer which is checked in the debounceChecks() which will call this from the main loop. I'll try to note if I come across that issue again as it may make sense to re-enable to ensure the looping effects get stopped.